etc/adminhtml/system.xml

```diff
@@ -920,6 +920,36 @@
                     </field>
                 </group>
 
+                <group id="omise_offsite_paypay" translate="label" sortOrder="535" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Enable PayPay QR Payment</label>
```

Comment:
Elsewhere we call this "PayPay Payment"

https://github.com/omise/omise-magento/blob/fcc3f8d07f1927f1cd153b8d467c1db51a65cb89/Helper/OmiseHelper.php#L206